periphery/tenderly/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
this can go since the action is not written in TS

Reply:
yeah, but it's the one that tenderly has autogenerated (likely as in your first approach). It's imo okay to keep it -> as said, I wasn't aware that this has been generated by tenderly in the first place.

Reply:
exactly